src/Apps/Artwork/Components/ArtworkAuctionCreateAlertHeader/ArtworkAuctionCreateAlertHeader.tsx

```diff
@@ -0,0 +1,49 @@
+import { Box, Text } from "@artsy/palette"
+import * as React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { ArtworkAuctionCreateAlertHeader_artwork$data } from "__generated__/ArtworkAuctionCreateAlertHeader_artwork.graphql"
+import { ArtworkSidebarCreateAlertButtonFragmentContainer } from "Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCreateAlertButton"
+import { useFeatureFlag } from "System/useFeatureFlag"
+
+interface ArtworkAuctionCreateAlertHeaderProps {
+  artwork: ArtworkAuctionCreateAlertHeader_artwork$data
+}
+
+const ArtworkAuctionCreateAlertHeader: React.FC<ArtworkAuctionCreateAlertHeaderProps> = ({
+  artwork,
+}) => {
+  const isArtworkAuctionCreateAlertHeaderEnabled = useFeatureFlag(
```

Comment:
re: https://github.com/artsy/force/pull/12678/files#r1282001013 , I think this should be moved up to the layout